renormalizer/spectra/finitet.py

```diff
@@ -36,7 +36,9 @@ def __init__(
         gs_shift=0,
         dump_dir: str=None,
         job_name=None,
+        nonradiativedecay=None
```

Comment:
The default value should be `False` to indicate that this is a Tur/False option.